src/styles/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $days-in-week * $day-size + $gap * ($days-in-week - 1);
+  gap: $gap;
+
+  @each $day, $num in $start-day {
+    &--start-day-#{$day}
+    &__day:first-child {
+      margin-left: ($num - 1) * $day-size + $gap * ($num - 1);
+    };
+  };
+
+  @for $daysCount from 28 through $days-total {
+    &--month-length-#{$daysCount}
+    &__day:nth-child(n + #{$daysCount + 1}) {
+      display: none;
+    };
+  };
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+
+    width: 100px;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid #000;
+
+    transition-duration: 500ms;
```

Comment:
For the future, specify to which properties should apply the transition.
And you forget to add `cursor: pointer`
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/04349574-7275-4246-abe2-9d450f26ef29)

Reply:
Will be more careful in future, thank you!